core/src/main/resources/swagger/rawls.yaml

```diff
@@ -2566,7 +2632,10 @@ paths:
           description: Rawls Internal Error
           schema:
             $ref: '#/definitions/ErrorReport'
-      description: Rename method configuration
+      description: |
+        Rename method configuration.
+        The workspace name and namespace in the HTTP body are ignored.
```

Comment:
I agree with all that.  But if we do ever see an API call with this conflict, the caller is doing something wrong, so we should respond with an error.